charts/namespaces/templates/namespaces.yaml

```diff
@@ -119,4 +119,26 @@ metadata:
   namespace: ingress
 type: Opaque
 data:
-  password: {{ .Values.secret.password | b64enc | quote }}
\ No newline at end of file
+  password: {{ .Values.secret.password | b64enc | quote }}
+
+---
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name:  api-gateway
+  annotations:
+    "helm.sh/resource-policy": keep
+    "linkerd.io/inject" : enabled
+    "config.linkerd.io/trace-collector": collector.linkerd-jaeger:55678 # or 14268?
+
+---
+
+apiVersion: v1
```

Comment:
this secret is interesting. If you look at 'values.yaml' it will set a default password. I did this for the CloudLab installation and will overwrite the secrets later. 

Hoewver, running locally it should give connections issues to rabbitMQ. I am honestly not sure why it is running on my machine witout errors...

Keep this secret here though, this just my thought process running